packages/ui/src/components/Alert/Alert.tsx

```diff
@@ -4,7 +4,8 @@ import { IconAlertTriangle } from '../Icon/icons/IconAlertTriangle'
 import { IconInfo } from '../Icon/icons/IconInfo'
 import { IconX } from '../Icon/icons/IconX'
 
-import styleHandler from '../../lib/theme/styleHandler'
+// @ts-ignore
+import useStyles from '../../lib/theme/use-styles'
```

Comment:
@saltcod Is this `@ts-ignore` necessary? Might've been from the merge conflict resolution, but this might clobber the types for the hook